src/components/analyses/listing/DeleteConfirmationDialog.js

```diff
@@ -0,0 +1,65 @@
+/**
+ * A confirmation dialog for deleting analyses.
+ *
+ * @author psarando
+ */
+import React from "react";
+import { useTranslation } from "i18n";
+
+import {
+    Button,
+    Dialog,
+    DialogActions,
+    DialogContent,
+    DialogContentText,
+    DialogTitle,
+    useMediaQuery,
+    useTheme,
+} from "@material-ui/core";
+
+import { build } from "@cyverse-de/ui-lib";
+
+import ids from "../ids";
+
+function DeleteConfirmationDialog({ baseId, open, onClose, confirmDelete }) {
```

Comment:
This is almost an exact copy of the `src/components/analyses/listing/MultiRelaunchWarningDialog.js`. The only difference being the base debug ID, dialog title and content text, and the OK button's `onClick` confirmation function.

Maybe these should be refactored to use a common, generic confirmation dialog.